cooperator_api_logs/security/ir.model.access.csv

```diff
@@ -1,2 +1,2 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_emc_api_log_group_system,access_emc_api_log_group_system,cooperator_api_logs.model_emc_api_log,base.group_system,1,0,1,0
+access_cooperator_api_log_group_system,access_cooperator_api_log_group_system,cooperator_api_logs.model_cooperator_api_log,base.group_system,1,0,1,0
```

Comment:
I am not certain whether this requires a migration step or not.

Reply:
Mmh I don't think so : can @remytms or @huguesdk confirm ?